apps/src/panels/PanelsView.tsx

```diff
@@ -126,6 +138,9 @@ const PanelsView: React.FunctionComponent<PanelsProps> = ({
     previousPanelIndex !== undefined &&
     panels[previousPanelIndex];
 
+  const nextPanel =
+    currentPanelIndex + 1 < panels.length && panels[currentPanelIndex + 1];
```

Comment:
nit: are both of these checks necessary? Any situation where `panels` would include `undefined` members?

Reply:
Oh, I'm using it like [this](https://stackoverflow.com/a/49723904/24055695) to retrieve the value from the array only if it should be valid. Not sure if it's the best pattern...